<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Bablu" failures="1" tests="4" name="Test Runner1" time="0.137" errors="0" timestamp="2024-11-08T13:12:15 IST">
  <testcase classname="distributedparallel.Test1" name="demo1" time="0.018"/>
  <testcase classname="distributedparallel.Test1" name="demo2" time="0.003"/>
  <testcase classname="distributedparallel.Test2" name="demo1" time="0.002"/>
  <testcase classname="distributedparallel.Test2" name="demo2" time="0.002">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at distributedparallel.Test2.demo2(Test2.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- demo2 -->
</testsuite> <!-- Test Runner1 -->
